homeassistant/core.py

```diff
@@ -1782,13 +1782,20 @@ def register(
             Coroutine[Any, Any, ServiceResponse] | None,
         ],
         schema: vol.Schema | None = None,
+        supports_response: SupportsResponse = SupportsResponse.NONE,
```

Comment:
Please break out the core change to a separate PR.

Reply:
I have rebased this PR after #106164 got merged.